frontend/src/services/objectService.ts

```diff
@@ -227,7 +228,54 @@ export default {
   searchObjects(params: SearchObjectsOptions = {}, headers: any = {},) {
     // remove objectId array if its first element is undefined
     if (params.objectId && params.objectId[0] === undefined) delete params.objectId;
-    return comsAxios().get(PATH, { params: params, headers: headers });
+
+    if (params.objectId) {
+      /**
+       * split calls to COMS if query params (eg objectId's)
+       * will cause url length to excede 2000 characters
+       * see: https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers
+       *
+       * TODO: consider creating a utils function
+       * eg: `divideParam(params, attr)`
+       *     ... return Promise.all(divideParam(params, objectId).map(zparam => comsAxios().get(PATH, {params: zparam, headers: headers});
```

Comment:
Break up the following comment line - it's just too long even for our permissive linter.
> This line has a comment length of 138. Maximum allowed is 120